### PR TITLE
Fi 1000 fix splitting scope with extra spaces

### PR DESCRIPTION
**Pull request by hershilpatel**
Reference server now allows each scope in a scope string to be separated by multiple spaces

### CODE REVIEW

**Review by Jammjammjamm — COMMENTED** [review]
When I visit: http://localhost:8080/reference-server/oauth/authorization?response_type=code&client_id=SAMPLE_CONFIDENTIAL_CLIENT_ID&redirect_uri=http%3A%2F%2Flocalhost%3A4567%2Finferno%2Foauth2%2Fstatic%2Fredirect&scope=+launch+launch%2Fpatient++++offline_access+openid+profile++user%2F*.*+patient%2F*.*+++&state=eb1c05fe-122f-4cd0-aac6-8535f14cd1a2&aud=http%3A%2F%2Flocalhost%3A8080%2Freference-server%2Fr4&patient_id=18

I see this:

Screen Shot 2020-10-05 at 10 04 23 AM

**Review by okeefm — CHANGES_REQUESTED** [review]
I'm seeing the same issue as @Jammjammjamm.

**Review by okeefm — APPROVED** [review]
Works as expected now. Looks good!

**Review by radamson — COMMENTED** [review]
Works great!  Left a couple minor code suggestions for your consideration.

File: src/main/java/org/mitre/fhir/utils/FhirReferenceServerUtils.java [hunk 1]
```diff
@@ -95,4 +97,54 @@ public static String createAuthorizationHeaderValue(String accessToken) {
...
+
+    List<String> scopesList = new ArrayList<>();
+
+    for (String scope : scopesArray) {
+      if (!scope.equals("")) {
+        scopesList.add(scope);
+      }
+    }
+
+    return scopesList;
```
[the remark this answers is not in the document]
Comment (radamson) [reply src/main/java/org/mitre/fhir/utils/FhirReferenceServerUtils.java:109]: What's the verdict on this? 👍 👎

**Review by Jammjammjamm — APPROVED** [review]

**Review by radamson — APPROVED** [review]

**Pull request merged**